deploy/01_manager.ts

```diff
@@ -40,5 +37,5 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
 func.id = "deploy_manager";
 func.tags = ["Manager", "core"];
-func.dependencies = noDependencies ? undefined : ["MultiSig"];
+// func.dependencies = ["MultiSig"];
```

Comment:
Should this dependency still be here? If yes, uncomment, if not, remove commented unnecessary code.